webapp/src/modules/catalog/reducer.ts

```diff
@@ -0,0 +1,69 @@
+import { CatalogItem } from '@dcl/schemas'
+import {
+  loadingReducer,
+  LoadingState
+} from 'decentraland-dapps/dist/modules/loading/reducer'
+import {
+  FETCH_CATALOG_FAILURE,
+  FETCH_CATALOG_REQUEST,
+  FETCH_CATALOG_SUCCESS,
+  FetchCatalogFailureAction,
+  FetchCatalogRequestAction,
+  FetchCatalogSuccessAction
+} from './actions'
+
+export type CatalogState = {
+  data: Record<string, CatalogItem>
+  loading: LoadingState
+  error: string | null
+}
+
+export const INITIAL_STATE: CatalogState = {
+  data: {},
+  loading: [],
+  error: null
+}
+
+type CatalogReducerAction =
+  | FetchCatalogFailureAction
+  | FetchCatalogRequestAction
+  | FetchCatalogSuccessAction
+
+export function catalogItemReducer(
```

Comment:
```suggestion
export function catalogReducer(
```
Can we remove the `item` from the name? I think this one looks more accurate, wdyt?